spec/latest/core/core.jsonld

```diff
@@ -97,15 +97,15 @@
       "@type": "hydra:Class",
       "subClassOf": "rdfs:Resource",
       "label": "Hydra Resource",
-      "comment": "The class of dereferenceable resources.",
+      "comment": "The class of dereferenceable resources by means a client can at least try to obtain a resource but it still should verify received payloads.",
       "vs:term_status": "testing"
     },
     {
       "@id": "hydra:Class",
       "@type": [ "hydra:Resource", "rdfs:Class" ],
-      "subClassOf": [ "hydra:Resource", "rdfs:Class" ],
+      "subClassOf": [ "rdfs:Class" ],
       "label": "Hydra Class",
-      "comment": "The class of Hydra classes. Hydra classes and their instances are dereferenceable resources.",
+      "comment": "The class of Hydra classes.",

```

Comment:
Should we explain what "The class of Hydra classes" actually mean? If a `hydra:Class` is no longer dereferenceable, what good does it do?

Reply:
Id' leave it as is. Most of the hydra terms are rooted in hydra:Class - I believe it's enough to have it just to have those hydra terms _separated_.

>>If a hydra:Class is no longer dereferenceable,

We indeed removed this requirements as it is virtually impossible to enforce, but still I'd read something that is a hydra:Class SHOULD be dereferencable (I don't see any good reasons against it). Developers still have a possibility of not having those resource dereferencable.